docs/distribution/standard-libraries.md

```diff
@@ -33,7 +33,7 @@ these libraries, as well as notes on how they should be used.
 `Base` is the core library of Enso. It contains core types and data structures,
 as well as basic functionality for interacting with the outside world. It can be
 found in
-[`distribution/std-lib/Standard/src/Base.enso`](https://github.com/enso-org/enso/tree/main/distribution/std-lib/Standard/src).
+[`distribution/lib/Standard/Base/0.1.0/src/Main.enso`](https://github.com/enso-org/enso/tree/main/distribution/lib/Standard/Base/0.1.0/src/Main.enso).
```

Comment:
I'm not entirely sure where these are supposed to point to as they were broken, but I understand it should be the Main.enso file of each library, correct? cc: @iamrecursion